.env.example

```
@@ -240,4 +240,7 @@ ENABLE_IMPORT_QUEUE=
 AMPLITUDE_API_KEY_SCIWEAVE=
 AMPLITUDE_SECRET_KEY_SCIWEAVE=
 
-APPLE_CLIENT_ID=
\ No newline at end of file
+APPLE_CLIENT_ID=
+REVENUECAT_WEBHOOK_SECRET=
+REVENUECAT_API_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REVENUECAT_API_KEY key should go before the REVENUECAT_WEBHOOK_SECRET key

(UnorderedKey)